test/export/testing.py

```diff
@@ -2,6 +2,196 @@
 import unittest
 from unittest.mock import patch
 
+import torch
+
+aten = torch.ops.aten
+
+COMPOSITE_OPS_THAT_CAN_BE_PRESERVED = [
```

Comment:
hmm... why do we want a hardcoded list here for testing? I'm worried that people will read this and expect that it is a source of truth, but it will inevitable go stale if/when someone adds a new CompositeImplicit op.